README.md

```diff
@@ -1 +1,74 @@
-# forta-challenge
\ No newline at end of file
+# Nethermind Forta Bot
+
+## Description
+
+This project implements a Forta bot to monitor bot deployments, updates, and deletions by Nethermind. The bot generates alerts whenever a bot is created, updated, or deleted, helping to ensure transparency and traceability of bot management activities.
+
+## Supported Chains
```

Comment:
Please keep only the relevant chain(s), on which this bot can be deployed. It's not all of those listed here (similarly, the `chainIds` should also then be updated in the `package.json`